File: app/src/main/java/com/example/austi/musiccontrol/MySettingFragment.java
package com.example.austi.musiccontrol;

import android.os.Bundle;
import android.preference.PreferenceFragment;

/**
 * Created by erinmunoz on 6/6/16.
 */
public class MySettingFragment extends PreferenceFragment
{
    @Override
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.fragment_preferences);
    }
}

File: app/src/main/kotlin/com/example/austi/musiccontrol/MySettingFragment.kt
package com.example.austi.musiccontrol

import android.os.Bundle
import android.preference.PreferenceFragment

class MySettingFragment : PreferenceFragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.fragment_preferences)
    }
}
